main/main.c

```diff
@@ -66,6 +66,8 @@ static void init_software();
 static inline SAT_returnState init_csp_interface();
 static void init_system_tasks();
 void vAssertCalled(unsigned long ulLine, const char *const pcFileName);
+static FTP app;
```

Comment:
What's this?

Reply:
this is Evan's FTP. Maybe we can give the variable a better name.